readme.md

```diff
@@ -1,7 +1,7 @@
 # HTML form
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+- [DEMO LINK](https://Taras0506.github.io/layout_html-form/)
+- [TEST REPORT LINK](https://Taras0506.github.io/layout_html-form/report/html_report/).
```

Comment:
The demo link from here is working. You should replace the link from the description with this one. But can't open the link with the tests. Make sure you run the npm test command